docs/tests.md

```diff
@@ -1,17 +1,22 @@
 # Tests for actors
 
-You can implement two types of tests for an actor in the leapp-repository - component and unit tests.
+The Leapp actors are currently covered by three types of tests - unit, component and e2e.
 
-## Component tests
-   - These tests provide fabricated input messages for the actor, check the outputs stated in the actor's description.
-   - These tests should not be written based on the actor's code but rather based on the behavior stated in the actor's description. Ideally they could be written by somebody who doesn't know the code.
-   - Example of a component test: https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/actors/redhatsignedrpmcheck/tests/test_redhatsignedrpmcheck.py
-
-## Unit tests
+## Unit and component tests
+- Both unit and component tests are to be placed in the actor's _tests_ folder.
+- Tutorial on how to write unit and component tests: https://leapp.readthedocs.io/en/latest/unit-testing.html
```

Comment:
Could we rename the "unit-testing" part of the URL? This is sort of misleading with current content.

Reply:
Yes, I agree. I thought about that. But I meant to change it on all the places in one push. In separate PRs.